css/styles.css

```diff
@@ -0,0 +1,282 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin-top: 0;
+}
+
+.flex-fix {
+  display: flex;
+  flex-direction: row;
+}
+
+.main-box {
+  margin: auto;
+}
+
+.upper-section {
+  max-width: 100%;
+  height: 40.652em;
+  background: url("../images/mount.jpg") 50% 100% no-repeat;
+  background-size: cover;
+}
+
+.top-box {
+  overflow-y: hidden;
+  width: 77%;
+  height: 100%;
+  padding: 6% 0 0 0;
+  margin: 0 auto;
+}
+
+.logo {
+  height: 1.8125em;
+}
+
+.top-menu {
+  list-style: none;
+  width: 94%;
+  height: 100%;
+  padding: 0;
+  margin: 0;
+}
+
+.top-menu li {
+  height: 100%;
+  width: 22%;
+  border-right: 0.0625em solid #262626;
+  box-shadow: 0 0 0.625em 0.03125em #a6a6a6;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+nav {
+  background: linear-gradient(to bottom, #f2f2f2, #b3b3b3 50%, #999 50%, #666);
+  height: 2.1875em;
+  max-width: 93%;
+  position: relative;
+  left: 27px;
+}
+
+.top-menu > li > a {
+  text-decoration: none;
+  text-align: center;
+  color: rgb(255, 240, 238);
+  text-shadow: 0.0625em -0.0625em 0.625em rgb(52, 48, 48);
+  font-size: 0.8125em;
+  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+.nav-form {
+  width: auto;
+  position: relative;
+  justify-content: flex-end;
+  align-items: center;
+  margin-right: -3.7em;
```

Comment:
- Your search bar is overflowing, you can adjust this line `margin-right: -3.7em;` to fix it